Optik/Classes/AlbumViewController.swift

```diff
@@ -226,12 +280,18 @@ internal final class AlbumViewController: UIViewController {
                 return nil
             }
             
-            return ImageViewController(image: images[index], index: index)
+            pageControl?.numberOfPages = images.count
+            
+            let imageViewController =  ImageViewController(image: images[index], index: index)
+            imageViewController.likeButton?.addTarget(self, action: #selector(AlbumViewController.didSelectImage(_:)), for: .touchUpInside)
+            return imageViewController
```

Comment:
The original code always return this view controller. I didn't change that. The only thing that I change was add the target for the button